### PR TITLE
Third batch of module tmpl codestyle

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Third batch of module tmpl codestyle

### Testing Instructions

Please test and review the following frontend modules using the testing sampledata

- mod_menu
- mod_random_image
- mod_realted_items

### Expected result

Still works

### Actual result

Works with codestyle problems

### Documentation Changes Required

none

### CODE REVIEW

File: modules/mod_menu/tmpl/default.php [hunk 1]
```diff
@@ -19,83 +19,49 @@
...
+		<?php endif; ?>
+		<li class=" <?php echo $class; ?>">
+			<?php if (in_array($item->type, array('separator', 'component', 'heading', 'url'))) : ?>
+				<?php require JModuleHelper::getLayoutPath('mod_menu', 'default_' . $item->type); ?>
+			<?php else : ?>
+				<?php require JModuleHelper::getLayoutPath('mod_menu', 'default_url'); ?>
+			<?php endif; ?>
+			<?php // The next item is deeper. ?>
+			<?php if ($item->deeper) : ?>
+				<?php echo '<ul class="nav-child unstyled small">'; ?>
```
Comment (Quy) [inline modules/mod_menu/tmpl/default.php]: Use HTML only

File: modules/mod_menu/tmpl/default.php [hunk 2]
```diff
@@ -19,83 +19,49 @@
...
+			<?php elseif ($item->shallower) : ?>
+				<?php // The next item is shallower. ?>
+				<?php echo '</li>'; ?>
```
Comment (Quy) [inline modules/mod_menu/tmpl/default.php]: Use HTML only

File: modules/mod_menu/tmpl/default.php [hunk 3]
```diff
@@ -19,83 +19,49 @@
...
+				<?php echo str_repeat('</ul></li>', $item->level_diff); ?>
+			<?php else : ?>
+			<?php // The next item is on the same level. ?>
+				<?php echo '</li>'; ?>
```
Comment (Quy) [inline modules/mod_menu/tmpl/default.php]: Use HTML only

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 90aef02dbc819e4bb7ce1dd178889848d4061ae6<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17855">issues.joomla.org/tracker/joomla-cms/17855</a>.</sub>

**Comment by zero-24** [comment]
can we have a second review & merged here?

**Pull request closed without merging**